Some tests are failing:

```
--- FAIL: TestSwitchNS (0.00s)
    --- FAIL: TestSwitchNS/no-op (0.00s)
        interpreter_test.go:112:
                Error Trace:    /build/source/internal/view/cmd/interpreter_test.go:112
                Error:          Not equal:
                                expected: "pod blee"
                                actual  : "pod fred"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -pod blee
                                +pod fred
                Test:           TestSwitchNS/no-op
    --- FAIL: TestSwitchNS/no-ns (0.00s)
        interpreter_test.go:112:
                Error Trace:    /build/source/internal/view/cmd/interpreter_test.go:112
                Error:          Not equal:
                                expected: "pod blee"
                                actual  : "pod"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -pod blee
                                +pod
                Test:           TestSwitchNS/no-ns
    --- FAIL: TestSwitchNS/happy (0.00s)
        interpreter_test.go:112:
                Error Trace:    /build/source/internal/view/cmd/interpreter_test.go:112
                Error:          Not equal:
                                expected: "pod app=blee @zorg blee"
                                actual  : "pod app=blee @zorg fred"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -pod app=blee @zorg blee
                                +pod app=blee @zorg fred
                Test:           TestSwitchNS/happy
FAIL
FAIL    github.com/derailed/k9s/internal/view/cmd       0.012s
FAIL
```

--- a/pkgs/by-name/k9/k9s/package.nix
+++ b/pkgs/by-name/k9/k9s/package.nix
@@ -12,13 +12,13 @@
 
 buildGoModule (finalAttrs: {
   pname = "k9s";
-  version = "0.50.7";
+  version = "0.50.9";
 
   src = fetchFromGitHub {
     owner = "derailed";
     repo = "k9s";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-K0JETHs2vOOfDUPw22x+2O5WN0rtkXsRxMnUHrCpkDg=";
+    hash = "sha256-wTQOBxJgrDPcWSiezCwHNgvfGa6oWBM+DNa7RC/9PJA=";
   };
 
   ldflags = [
@@ -32,7 +32,7 @@ buildGoModule (finalAttrs: {
 
   proxyVendor = true;
 
-  vendorHash = "sha256-6Xyvgy+V0vLCqMgkUR+n6M6wGs7eU78ckLCKF9WrI2w=";
+  vendorHash = "sha256-SDl47tAYiE00gFpCvoBeiV8XL/E29457b1RQW3pogmM=";
 
   # TODO investigate why some config tests are failing
   doCheck = !(stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64);

